docs/Cloud_integrations/Azure_IoT/Azure_IoT_Hub/Configure_Azure_IoT_Hub_integration.md

```diff
@@ -47,3 +51,51 @@ Use the obtained credentials to establish the integration between {{ coiote_shor
       - click **Save** to keep the setting.
 
 Your integration should now be established. To get the integration up and running, try [synchronizing devices with Azure IoT Hub](Device_operations/Synchronize_devices_with_Azure_IoT_Hub.md)
+
+
+## Configuring Azure resources via CLI
+
+This section illustrates the configuration of Azure resources through the Command-Line Interface (CLI), offering a streamlined and scriptable approach to resource management. It is assumed that you have completed the following prerequisites:
+
+1. **Installed Azure CLI:**
+   Ensure that Azure CLI is installed on your system. If not, refer to the [official installation guide](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli) for detailed instructions.
+
+2. **Authenticated to Azure CLI:**
+   Authenticate your Azure CLI by following the steps outlined in [this guide](https://learn.microsoft.com/en-us/cli/azure/authenticate-azure-cli-interactively).
+
+3. **Created Resource Group `coiote-dm-experiments`:**
```

Comment:
imo you should add a suggestion of the name, e.g. `Created Resource Group e.g. `coiote-dm-integration``, also I'm suggesting to change `coiote-dm-integration` to `coiote-dm-integration`

Reply:
ok